infra/src/main/kotlin/fi/fta/geoviite/infra/inframodel/InfraModelController.kt

```diff
@@ -234,7 +232,7 @@ private fun getInfraModelBatchFilenameLocalizationParams(
         mutableMapOf(
                 "applicability" to translation.t("enum.PlanApplicability.${applicability?.name ?: "UNKNOWN"}"),
                 "alignmentName" to alignmentName,
-                "date" to LocalDate.now().format(ZIP_TIMESTAMP_FORMATTER),
+                "date" to FILENAME_DATE_FORMATTER.format(Instant.now()),
```

Comment:
Tässä huomaa yhden eroavaisuuden aiempaan verrattuna: Haetaan aika `Instant.now()`:lta `LocalDate.now()`:n sijaan. `Instant`:in kanssa `DateTimeFormatter` osaa ottaa aikavyöhykkeet huomioon, joten tiedostonimien aikaleimojen pitäisi tulla nyt aina Helsingin mukaan